Modules/Microsoft365DSC/DSCResources/MSFT_O365SearchAndIntelligenceConfigurations/settings.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+    "resourceName": "O365SearchAndIntelligenceConfigurations",
+    "description": "",
```

Comment:
Please match this description to the Description in the Readme file

Reply:
Humm, we should look at a way not to duplicate this work in the future.